shell/pages/c/_cluster/longhorn/index.vue

```diff
@@ -3,42 +3,59 @@ import { mapGetters } from 'vuex';
 
 import InstallRedirect from '@shell/utils/install-redirect';
 
+import { SERVICE } from '@shell/config/types';
 import { NAME, CHART_NAME } from '@shell/config/product/longhorn';
 
+import IconMessage from '@shell/components/IconMessage';
 import LazyImage from '@shell/components/LazyImage';
+import Loading from '@shell/components/Loading';
 
 export default {
-  components: { LazyImage },
+  components: {
+    IconMessage, LazyImage, Loading
+  },
 
   middleware: InstallRedirect(NAME, CHART_NAME),
 
+  async fetch() {
+    if ( this.$store.getters['cluster/schemaFor'](SERVICE) ) {
+      await this.$store.dispatch('cluster/findAll', { type: SERVICE });
```

Comment:
We can use `findMatching`, but it was broken - this PR fixes it  https://github.com/rancher/dashboard/pull/10413